contracts/javascore/xcall/src/intTest/java/ibc/xcall/integration/CSIntegrationTest.java

```diff
@@ -0,0 +1,119 @@
+///*
```

Comment:
Do we need to keep these commented files?

Reply:
deleted unused/commented files